### DISCUSSION TITLE
No playbook Alerts showing in Kibana/Dashboards

### DISCUSSION THREAD

**Discussion by THExCAPN** [Security-Onion-Solutions/securityonion#12687] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

64

### RAM

800GB

### Storage for /

250GB

### Storage for /nsm

17TB

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

About a few day ago I noticed I was no longer getting any alerts from Playbook/elastalert. I have not made any configuration changes for a couple weeks prior to when it stopped working. I am however still getting logs.

Some interesting behavior is when I use the elastalert test and select a rule I know should have at least one hit, it will come back with 0 but still say the test completed successfully.

So far I have looked in the following log sources and non of them have anything that would indicate there is an issue.
```
/opt/so/log/elastalert/elastalert.log
/opt/so/log/elasticsearch/securityonion.log
/opt/so/log/playbook/playbook.log
```

Any guidance on where else I could look to resolve this would be appreciated.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by InfosecGoon** [comment]
Do you get results if you manually query Elasticsearch with the query in your Elastalert rule?

Try this from Kibana Dev Tools:

```
GET /*/_eql/search
{ "query": """

$QUERYGOESHERE

"""}
```

**Reply by THExCAPN** [reply]
Thanks for the response,

I was  actually figured out what went wrong, It had to do with the playbook index itself. It somehow started using one of my custom pipelines as the default. Once I got that sorted the alerts started showing up properly. 

For anyone with the issue in the future, I was able to see this by going to Kibana > Stack Management > Index Management > Turning on "Include hidden indices" and then searching for the latest playbook.alerts index. From there you can see all the settings the index is using.
